src/context/theme.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {
+  createContext,
+  FunctionComponent,
+  useCallback,
+  useContext,
+  useState,
+} from "react";
+import {
+  CustomTheme,
+  govWalletTheme,
+  sallyTheme,
+} from "../common/styles/themes";
+
+interface ThemeContext {
+  theme: CustomTheme;
+  setTheme: (themeName: string | undefined) => void;
+}
+
+export const ThemeContext = createContext<ThemeContext>({
+  theme: sallyTheme,
+  setTheme: () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+});
+
+export const ThemeContextProvider: FunctionComponent = ({ children }) => {
+  const [theme, setThemeValue] = useState(sallyTheme);
+
+  const setTheme = useCallback(
+    (themeName?: string) => {
+      if (themeName === "GOVWALLET") {
```

Comment:
Just a minor nit: Would it be better if `DEFAULT` and `GOVWALLET` were global constants?

Reply:
yup! changed [here](https://github.com/rationally-app/mobile-application/pull/493/commits/c4c55ee6d802eaf0e74c62b715427f7a90f889ce)!